### ISSUE TITLE
Editor 6 OS X — ALT + Delete not deleting whole words anymore

### ISSUE THREAD

**Issue by drreen** [GitbookIO/feedback#72]
Gitbook Editor used to delete a whole word when using ALT+Delete.Now it's doing nothing.
Only CMD+Delete does that

About 99% of the apps out there incl all browsers are going with this:
ALT+Delete: Delete whole word
CMD+Delete: Delete whole line

I suggest sticking to those standards.
Please add it to next hotfix :)

**Comment by drreen** [comment]
Also not working in raw mode :)

**Comment by SamyPesse** [comment]
It'll be fixed in today release: `ALT+Delete` will now also delete the whole word.

On mac the follow will be equivalent:

```
Ctrl-H = Backspace
Alt-Backspace = Cmd-Backspace
Ctrl-D = Delete
Ctrl-Alt-Backspace = Cmd-Delete
Alt-Delete = Cmd-Delete
Alt-D = Cmd-Delete
```

**Comment by SamyPesse** [comment]
Fixed in 6.0.3
